fire/SFMainMod.F90

```diff
@@ -288,306 +344,126 @@ subroutine CalculateSurfaceRateOfSpread(currentSite)
   end subroutine CalculateSurfaceRateOfSpread
   
   !---------------------------------------------------------------------------------------
+  
+  subroutine CalculateSurfaceFireIntensity(currentSite)
+    !
+    !  DESCRIPTION:
+    !  Calculates surface fireline intensity for each patch of a site
+    !  Right now also calculates the area burnt...
+    !
+    use FatesConstantsMod, only : m2_per_km2
+    use SFEquationsMod,    only : FireDuration, LengthToBreadth
+    use SFEquationsMod,    only : AreaBurnt, FireSize, FireIntensity
+    use SFParamsMod,       only : SF_val_fire_threshold
 
-  !*****************************************************************
-  subroutine  ground_fuel_consumption ( currentSite ) 
-  !*****************************************************************
-    !returns the  the hypothetic fuel consumed by the fire
-      use SFParamsMod, only: SF_val_mid_moisture, SF_val_mid_moisture_Coeff, SF_val_mid_moisture_Slope
-      use SFParamsMod, only : SF_val_min_moisture, SF_val_low_moisture_Coeff, SF_val_low_moisture_Slope
-      use SFParamsMod, only : SF_val_miner_total
-
-    type(ed_site_type) , intent(in), target :: currentSite
-    type(fates_patch_type), pointer    :: currentPatch
-    type(litter_type), pointer      :: litt_c           ! carbon 12 litter pool
+    ! ARGUMENTS:
+    type(ed_site_type), intent(inout), target :: currentSite
     
-    real(r8) :: moist           !effective fuel moisture
-    real(r8) :: tau_b(num_fuel_classes)     !lethal heating rates for each fuel class (min) 
-    real(r8) :: fc_ground(num_fuel_classes) !total amount of fuel consumed per area of burned ground (kg C / m2 of burned area)
-    integer :: tr_sf, tw_sf, dl_sf, lg_sf
-    integer  :: c
+    ! LOCALS:
+    type(fates_patch_type), pointer :: currentPatch                    ! patch object
+    real(r8)                        :: fuel_consumed(num_fuel_classes) ! fuel consumed [kgC/m2]
+    real(r8)                        :: tree_fraction_patch             ! treed fraction on patch [0-1]
+    real(r8)                        :: length_to_breadth               ! length to breadth ratio of fire ellipse (unitless)
+    real(r8)                        :: fire_size                       ! size of fire [m2]
+    real(r8)                        :: area_burnt                      ! area burnt [m2/km2]
```

Comment:
`tree_fraction_patch`, `length_to_breadth`, `fire_size`, and `area_burnt` are no longer used.